<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadExample</class>
 <widget class="QMainWindow" name="ThreadExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>7</y>
      <width>466</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Starting Folder </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_start_folder"/>
     </item>
     <item>
      <widget class="QPushButton" name="bt_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="te_output">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>71</y>
      <width>607</width>
      <height>222</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>ThreadExample</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_start</sender>
   <signal>clicked()</signal>
   <receiver>ThreadExample</receiver>
   <slot>start_listing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ThreadExample</receiver>
   <slot>cancel_listing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_listing()</slot>
  <slot>cancel_listing()</slot>
 </slots>
</ui>
